app/src/component/AppNavigator.js

```diff
@@ -122,6 +131,13 @@ export const AppNavigator = new StackNavigator({
 class AppWithNavigationState extends React.Component {
   static supportedDisplayTypes = ['toast'];
 
+  constructor() {
+    super();
+    this.state = {
```

Comment:
You can define this without a constructor
https://hackernoon.com/the-constructor-is-dead-long-live-the-constructor-c10871bea599